### DISCUSSION TITLE
[suggestion]Add more information to the Wiki Dialogue Codig.

### DISCUSSION THREAD

**Discussion by howltrek** [Monika-After-Story/MonikaModDev#9603] [QA - code and content creation (technical)] [answered]
I think there are many important explanations missing on how to code dialogue for Monika.

for example, everything that can be done with
tags:

labelthis:
jump this
etc.

Also on how to make different versions of a dialogue, according to information given by the player.
gender, age, eye color, etc.

Or block dialogs, so they only appear if certain events have occurred.
The first kiss for example.

or commands that make Monika perform certain actions like kisses.

**Comment by Booplicate** [comment]
It seems to me you're looking for programming or renpy tutorials/examples. That's not really related to this project, you can find that information online.

For something specific like kisses, you can find examples in the code, there's plenty of comments explaining how things work.

**Comment by ghost** [comment, marked as answer]
me and my friends are working on tutorials for MAS. you can check the progress at our organization, [Friends of Monika](https://github.com/Friends-of-Monika)!

**Comment by multimokia** [comment]
Once these are finished, you're welcome to contribute these to the wiki as well as utilties. They seem like they'd really help new contributors/developers get into the swing of things, especially seeing the CI and MDK items there. Hopefully we wind up in a place where we get safer submods (i.e. less potential for upstream flow break/crashes)

Putting a pin in that to potentially see about codesense way down the line to help catch things i.e. `_mas_` prefixes for custom vars (should not be used as is not official MAS and may conflict)

Likewise things that should be checked (first kiss as an example of a common flowbreak)

**Comment by ghost** [comment]
deal! we'll be working on it further, to make things presentable for a contribution. 💛
